packages/blockchain-wallet-v4-frontend/src/scenes/Dex/components/InfoWidget/InfoWidget.tsx

```diff
@@ -12,14 +12,15 @@ const Description = styled(Text)`
 
```

Comment:
Same component as it was, just moved to an upper level + width prop is added for an image